core/flow/property_handler.go

```diff
@@ -0,0 +1,32 @@
+package flow
+
```

Comment:
Could you please rebase the newest code?
We add the new helper.go in ext/datasource.
The Converter and Updater for property had been merged into master.
You might need to remove duplicated code.
Thanks

Reply:
done

Reply:
Please check the changes of handler, i think when the data is nil, we should call the *.ClearRules.If there is anything wrong with my understanding, please let me know.

Reply:
In our design, `LoadRules(nil)` is equivalent to `ClearRules()`.

Reply:
I know, but in yuting's commit , when the data is nil , it return directly, i think before return , we need call ClearRules() or LoadRules(nil).